docs/reference/search-connectors/release-notes.md

```diff
@@ -0,0 +1,34 @@
+---
+navigation_title: "Release notes"
```

Comment:
```suggestion
navigation_title: "Release notes"
mapped_pages:
  - https://www.elastic.co/guide/en/elasticsearch/reference/8.18/es-connectors-release-notes.html
```

We should add a mapped_page with full URL so version selector will have 8.x link too :)